commerce_coordinator/apps/lms/tasks.py

```diff
@@ -18,9 +19,13 @@
 User = get_user_model()
 
 
-class CourseEnrollTaskAfterReturn(Task):
+class CourseEnrollTaskAfterReturn(Task):    # pylint: disable=abstract-method
+    """
+    Base class for fulfill_order_placed_send_enroll_in_course_task
+    """
+
     def on_failure(self, exc, task_id, args, kwargs, einfo):
-        error_message = json.loads(exc.response.text).get('message', '')
+        error_message = json.loads(exc.response.text).get('message', '') if exc.response else 'Unknown error'
```

Comment:
change 'Unknown error' string to something else and then try. Also add doc string for this function, may it hels.

Reply:
nope. it didn't solve the docs issue